Makefile

```diff
@@ -93,3 +93,10 @@ clean-docker-images:
 		echo "<noop>";\
 	fi
 
+net:
+	tunctl -t tap0 -u `whoami`
+	brctl addif docker0 tap0
```

Comment:
Probably `docker0` could be replaced with a dynamic lookup of the bridge interface name `docker network inspect -f {{.Options}} bridge`. Although it still wouldn't be a fully universal solution, it'd make this target more universal.

Additionally, the bridge's interface name could become a variable with the default name derived from Docker's default bridge interface. In such case, a user would be able to override it via an environment variable.

Reply:
replaced for ip and a variable